### PR TITLE
fix: add Saurabh Mehta

### PR DESCRIPTION
**Pull request by saurabhmehta1601**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/771"></a>

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Should be JSON file

**Comment by saurabhmehta1601** [comment]
I have corrected the syntax error in json file as well as renamed the file .

File: public/data/saurabhmehta1601.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "bio": "Javascript Full Stack Developer ",
+  "avatar":"https://github.com/saurabhmehta1601.png",
+  "links":[
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/saurabhmehta1601",
+      "icon": "github"
+    },
+    {
+    "name": "Follow me on Twitter",
```
Comment (krishguptadev) [inline public/data/saurabhmehta1601.json:12]:
```suggestion
      "name": "Follow me on Twitter",
```

File: public/data/saurabhmehta1601.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://twitter.com/saurabh__mehta",
```
Comment (krishguptadev) [inline public/data/saurabhmehta1601.json:13]:
```suggestion
      "url": "https://twitter.com/saurabh__mehta",
```

File: public/data/saurabhmehta1601.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "icon": "twitter"
```
Comment (krishguptadev) [inline public/data/saurabhmehta1601.json:14]:
```suggestion
      "icon": "twitter"
```

File: public/data/saurabhmehta1601.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Saurabh Mehta",
+  "bio": "Javascript Full Stack Developer ",
+  "avatar":"https://github.com/saurabhmehta1601.png",
+  "links":[
```
Comment (Panquesito7) [inline public/data/saurabhmehta1601.json:5]:
```suggestion
  "links": [
```

File: public/data/saurabhmehta1601.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Saurabh Mehta",
+  "bio": "Javascript Full Stack Developer ",
```
Comment (Panquesito7) [inline public/data/saurabhmehta1601.json:3]:
```suggestion
  "bio": "JavaScript Full Stack Developer",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/saurabhmehta1601.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/saurabhmehta1601",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/saurabh__mehta",
+    "icon": "twitter"
```
Comment (Panquesito7) [inline public/data/saurabhmehta1601.json:14]: *Suggested by @krishguptadev previously; just to bump and remind.*
```suggestion
      "icon": "twitter"
```

File: public/data/saurabhmehta1601.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Saurabh Mehta",
```
Comment (Panquesito7) [inline public/data/saurabhmehta1601.json:2]:
```suggestion
  "name": "Saurabh Mehta",
  "type": "personal",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**